torch/csrc/autograd/variable.h

```diff
@@ -1,6 +1,10 @@
 #pragma once
 
+#ifndef NO_PYTHON
 #include <Python.h>
+#else
```

Comment:
Couldn't we just always use python_stub?

Reply:
Done